services/catarse/spec/models/concerns/project/custom_validators_spec.rb

```diff
@@ -69,4 +69,70 @@
     end
   end
 
+  describe '#no_base64_images' do
+    let(:project) { build(:project) }
+
+    subject { project.valid? }
+
+    context 'when about_html has base64 images' do
+      before do
+        project.about_html = "<img src='data:image/png;base64'></img>"
+        project.save
+      end
+
+      it do
+        is_expected.to be_falsey
+      end
+
+      it 'adds error message to about_html'  do
+        expect(project.errors[:about_html]).to include I18n.t('activerecord.errors.models.project.attributes.about_html.invalid')
+      end
+    end
+
+    context 'when about_html hasn\'t base64 images' do
+      before do
+        project.about_html = "<img src='image.png'></img>"
+        project.save
+      end
+
+      it do
+        is_expected.to be_truthy
+      end
+
+      it 'don\'t add error message to about_html' do
+        expect(project.errors[:about_html]).not_to include I18n.t('activerecord.errors.models.project.attributes.about_html.invalid')
```

Comment:
Os matchers to rspec aceitam que tu use `not_to` ou `to_not`, pra ficar mais legível nesses casos, é melhor o `to_not`.